AmazonChimeSDK/AmazonChimeSDKTests/audiovideo/DefaultAudioVideoControllerTests.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 @testable import AmazonChimeSDK
+@testable import AmazonChimeSDKMedia
```

Comment:
What variables/methods are used in this package?

Reply:
Seems none being used. Remove this package.